libraries/botbuilder-dialogs-adaptive-teams/tests/actionTests.test.ts

```diff
@@ -122,11 +122,7 @@ describe('Actions', function () {
     ComponentRegistration.add(new AdaptiveComponentRegistration());
     ComponentRegistration.add(new TeamsComponentRegistration());
 
-    const resourceExplorer = new ResourceExplorer().addFolder(
-        path.join(__dirname, '../../tests/tests/actionTests'),
-        true,
-        false
-    );
+    const resourceExplorer = new ResourceExplorer().addFolder(path.join(__dirname, './actionTests'), true, false);
```

Comment:
Since you're touching this line - `path.join` parameters should be omitting the path join character. This should be `path.join(__dirname, 'actionTests')`. Part of what `path.join` does is use the proper path join character based on the platform. It does normalize the resulting path as well, but its purpose is more clear when you pass in separate path parts.

Reply:
🤦‍♂️